src/schemas/ProgramDonorSummary/resolvers/summaryStats.ts

```diff
@@ -180,6 +190,8 @@ const programDonorSummaryStatsResolver: (
       : 0,
     allFilesCount: aggregations.allFilesCount.value,
     filesToQcCount: aggregations.filesToQcCount.value,
+    donorsInvalidWithCurrentDictionaryCount:
+      aggregations.donorsInvalidWithCurrentDictionary.doc_count,
```

Comment:
dont know if it matters, could add the zero value for new aggregation here https://github.com/icgc-argo/platform-api/blob/64d157e3b95f2875096a7858030c07963e40254a/src/schemas/ProgramDonorSummary/resolvers/summaryStats.ts#L163